core/src/main/java/com/digitalpebble/storm/crawler/parse/filter/SubDocumentsParseFilter.java

```diff
@@ -0,0 +1,41 @@
+package com.digitalpebble.storm.crawler.parse.filter;
```

Comment:
This file is missing the license header. It is also not very useful in practice but is a good illustration of use.
Instead, what about adding it to the [parse test package](https://github.com/DigitalPebble/storm-crawler/tree/master/core/src/test/java/com/digitalpebble/storm/crawler/parse/filter) and use that for checking that we are getting what we expect?